gst/nnstreamer/tensor_filter/tensor_filter_support_cc.cc

```diff
@@ -102,30 +117,65 @@ void tensor_filter_subplugin::cpp_close (const GstTensorFilterProperties * prop,
 /**
  * @brief C V1 tensor-filter wrapper callback function, "invoke"
  */
-int tensor_filter_subplugin::cpp_invoke (const GstTensorFilterProperties *prop,
-    void *private_data, const GstTensorMemory *input,
-    GstTensorMemory *output)
+int tensor_filter_subplugin::cpp_invoke (const GstTensorFilterFramework * tf,
+    const GstTensorFilterProperties *prop, void *private_data,
+    const GstTensorMemory *input, GstTensorMemory *output)
 {
   start_up (obj, private_data);
```

Comment:
Looks Good ! 🤗
Minor suggestion: Here and other two lines in this file, how about replace `start_up (obj, private_data)` with
```
tensor_filter_subplugin *obj;
_start_up (obj, private_data);
```
and remove the macro `start_up` ?

Reply:
Applied a static-inline function and removed both marco definitions.